spec/spec_helper_local.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```

Comment:
is this because facter 4 breaks things?

Reply:
The facterdb gem will use facts from whatever is local facter installed which is still 2.x but if you specify a version like we do in sensu module, you get different fake facts. Facter 4 does break things.

Reply:
This probably needs a comment that explains that it is pinned so that facter 3 is used and when facter 4 is stable and stop breaking things, it can be removed.

Reply:
But we never want to test against facter 4 until puppet-agent uses it at which point we'd have to do other updates anyways. Not much value in calling this out because facter 4 isn't actually valid as a puppet version of facter that I'm aware of.  This version of facter is just which version of facts to pull from facterdb, doesn't actually matter which version of facter is used.  The issue with facter 4 is else where in tests.

Reply:
This looks like we have special configuration with an arbitrary version number in it that provides some magic to make things work. It just needs to have a comment explaining why it is set.

Reply:
Added a comment that this overrides detected gem version of facter and is specific to facterdb.